src/nih_wayfinding/app/styles/layout/_header-footer.scss

```diff
@@ -42,4 +42,5 @@ footer {
 
 .navbar-actionheader-subtitle {
     padding-right: 11rem;
+    color: #fff;
```

Comment:
@jfrankl made this match the color for the navbar title since the medium grey was very difficult to see on the blue